app/interactors/sanitise_params.rb

```diff
@@ -1,10 +0,0 @@
-# frozen_string_literal: true
-
-class SanitiseParams
-  include Interactor
-
-  def call
-    context.age = context.age.to_i
```

Comment:
I should have deleted this interactor after moving quote creation to the start of the chain. It is no longer needed.